Refresh.GameServer/Types/Assets/AssetSafetyLevel.cs

```diff
@@ -5,15 +5,20 @@ public enum AssetSafetyLevel
     /// <summary>
     /// This asset is used in normal gameplay/operation and is okay to be uploaded.
     /// </summary>
+    /// <seealso cref="SafeMedia"/>
     Safe = 0,
     /// <summary>
+    /// This asset is still used in normal gameplay similar to <see cref="Safe"/>, and is okay to be uploaded, but the distinction can be useful.
```

Comment:
this comment is hard to follow and doesnt actually explain what it means by media

Reply:
I got ChatGPT to do it because I'm tired